packages/manager/apps/telecom/src/app/telecom/pack/migration/offers/pack-migration-offers.html

```diff
@@ -131,10 +131,34 @@
                                     data-title="{{ offer.description }}"
                                     data-ng-repeat="offer in $ctrl.process.migrationOffers.result.offers track by $index"
                                 >
-                                    <span
-                                        data-ng-if="offer.modemRental"
-                                        data-ng-bind-html="offer.modemRental.text | tucFormatPrice"
-                                    ></span>
+                                    <label class="oui-select">
+                                        <select
+                                            class="oui-select__input"
+                                            data-ng-model="offer.modem"
+                                            name="modem"
```

Comment:
use the const variable instead.
Related to this: 
https://github.com/ovh/manager/pull/8776/files#r1088780563

Reply:
Nope, here it's the value of the field 'modem' not the string 'modem'. The value of the field 'modem' should be: yes, recycled or no.